artifacttemplates/http%3A%2F%2Fopentosca.org%2Ftest%2Fapplications%2Fartifacttemplates/App-A_connectTo-w1-wip1/files/connectTo.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
ist zwar nen kleines Script, denke aber für die Projekte wäre es wohl am ende am besten wenn ihr es in python macht, allein wegen fehleranfälligkeiten

Reply:
Also für CSARs die ihr dann im Projekt braucht, nicht hier jetzt ;D